### PR TITLE
Make content wider, Update some pages

### PR DESCRIPTION
**Pull request by m1ga**
* increase content width from 740px to 900px (wider makes it a bit harder to read, lines are too long)
* fixed https://titaniumsdk.com/guide/Alloy_Framework/Alloy_Guide/Alloy_Concepts.html#alloy-and-the-titanium-sdk table
* removed some `warning` blocks
* removed some `download` images (those big green down arrows)
* cleaned up Compatibility matrix
* removed version numbers from https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Prerequisites/ and linked to Compatibility matrix right away. So we only have to maintain one list of version numbers
* updated links in https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Guide/Welcome_To_Titanium/Titanium_Resources.html

To see the real page you can use the `Preview rich diff` (the :page_facing_up: icon on the top right side of the file view) to see the differences in markdown mode.

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_Advanced_Tools/README.md [hunk 1]
```diff
@@ -27,20 +25,6 @@ The Android NDK is required to build native Java/Kotlin modules for Android or t
...
-Titanium supports NDK Revision 11c and higher.
-
-It's recommended to download the NDK via Android Studio. This will install it under the directory the Android SDK is installed at. Installing it via Android Studio avoids macOS Catalina (and newer versions) from requesting your permission to use the NDK tools when doing a build.
-
-Alternatively, you can install the Android NDK to your own custom location by downloading from the following website:
-
-![download_05](/images/guide/download/attachments/29004836/download_05.png)
-
-[Download the Android NDK on developer.android.com](http://developer.android.com/sdk/ndk/index.html)
+If you build a module for the first time the CLI will automatically download a JDK version for you.
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_Advanced_Tools/README.md]: NDK, not JDK :)

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_SDK_Open_Source_Attribution_Notice.md [hunk 2]
```diff
@@ -4,7 +4,3 @@ weight: '100'
 ---
 
 # Titanium SDK Open Source Attribution Notice
-
-Up-to-date information is available at [http://www.appcelerator.com/opensource/](http://www.appcelerator.com/opensource/).
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_SDK_Open_Source_Attribution_Notice.md:8]: At this point, just delete this file and any incoming links.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Welcome_To_Titanium/Titanium_Resources.md [hunk 3]
```diff
@@ -11,31 +11,29 @@ Appcelerator strives to make the developer experience as effective and efficient
 
 ## Contents
 
-### Developer resources
+### Developer resources, Help and support
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Welcome_To_Titanium/Titanium_Resources.md]:
```suggestion
### Developer Resources, Help, and Support
```

**Comment by m1ga** [comment]
Updated the last parts:
* removed all gperf/git links and pages
* jdk -> ndk mistake
* adjusted the headline on the support page

**Review by cb1kenobi — APPROVED** [review]

**Pull request merged**